pom.xml

```diff
@@ -547,9 +549,14 @@
         </dependency>
         <dependency>
             <groupId>org.geotools</groupId>
-            <artifactId>gt-wfs</artifactId>
+            <artifactId>gt-opengis</artifactId>
             <version>${geotools.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-wfs</artifactId>
```

Comment:
Please add a brief XML comment explaining what this is for (why it was added) and why it has a different version number than Geotools proper.

Reply:
Indeed, I should do that here. Perhaps I should go back entirely to WIP because I should perhaps start a POM cleanup